conference/submissions.md

```diff
@@ -4,7 +4,51 @@ layout: default
 
 # Submit a proposal
 
-We welcome any type of proposal for this conference.  This is the
+
+### COVID-19 statement
+
+The organisers hope to be able to proceed with running
+the Nordic-RSE conference, as planned, in Stockholm on December 1-2.
+However, with the ongoing COVID-19 situation, we are aware that this relies on
+there being a continuing improvement in the situation over the coming months.
+It is also likely that social distancing measures will have some effect on our
+plans for the conference and full details will be advised nearer to the
+conference. We encourage you to go ahead with making a submission to the
+conference but urge you not to make any travel bookings that are non-refundable
+at this stage ([COVID-19 decision timeline](/conference/timeline/)).
+
+
+### Call for proposals
+
+Are you interested in research software and computational science? Do you
+prioritize the sustainability and reproducibility of research outputs? Do you
+build software to support research? Do you work as part of a team alongside
+people who build research software? Maybe you already have an official role as
+a Research Software Engineer (RSE) within your organisation or perhaps you are
+a researcher or academic? 
+
+Whatever your interest or involvement in the world of research software, the
+1st Nordic RSE Conference, which takes place on the 1st-2nd December 2020 in
+Stockholm, Sweden has something to offer you. Why not consider submitting an
+abstract for a presentation, poster or mini-workshop session to Nordic-RSE
+2020?
+
+In common with the many regional and national research software engineering
+workshops and conferences that have started to take place in recent years,
+Nordic RSE will be different to regular academic conferences. 
+
+We are a friendly, diverse and welcoming community of technical and research
+professionals and the conference will provide attendees with a great
+opportunity to meet and network with peers, learn new skills and participate in
+a range of seminar and discussion sessions. Whether you’re a regular conference
+attendee or have never been to a conference before, Nordic-RSE 2020 has
+something to offer you.
+
+
+### Proposal types
+
+We welcome any type of proposal for this conference but our forms will be structured
+for five categories (see below). This is the
```

Comment:
Not saying that something needs to be done now, but this would seem to increase the barrier for other types of events such as more panels, discussions, etc.  (they only fit into "mini-workshop".  I'm not sure what to do, since the types listed make it much easier to plan and schedule...

Reply:
I can maybe add a 6th category which is more freestyle and where I list other options? This is really good comment and I want to make this really open.

Reply:
I will merge and open an issue so that we can think there.

Reply:
#96